### PR TITLE
New MV

### PR DESCRIPTION
**Pull request by mullermp**
Bumps to V3 of the SDK
Removes Rails (moves functionality to aws-sdk-rails)
Fix some bugs

### CODE REVIEW

File: lib/aws/session_store/dynamo_db/version.rb [hunk 1]
```diff
@@ -1,21 +1,7 @@
...
-# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
-# ANY KIND, either express or implied. See the License for the specific
-# language governing permissions and limitations under the License.
-
-
 module Aws
   module SessionStore
     module DynamoDB
-      VERSION = "1.0.0"
+      VERSION = "2.0.0"
```
Comment (alextwoods) [inline lib/aws/session_store/dynamo_db/version.rb:4]: Load this from the file to reduce duplication here.

**Review by alextwoods — COMMENTED** [review]
Looks good overall - but looks like specs are broken. Credentials aren't getting loaded in the spec_helper.  Do these tests rely on aws creds in the env and create an actual ddb table?

**Comment by mullermp** [comment]
Tests pass locally. I do think they are doing "integration" tests using my credentials. I need to take a look at the tests more.

File: lib/aws/session_store/dynamo_db/version.rb (hunk 1 above)
Comment (mullermp) [reply lib/aws/session_store/dynamo_db/version.rb:4]: In our other gems it's duplicated, I think in the SDK case it's automated to change.

**Pull request merged**